.github/workflows/documentation.yml

```diff
@@ -27,6 +27,9 @@ jobs:
       - name: Install git repository optional dependencies
         run: |
           python3 -m pip install https://github.com/odlgroup/odl/archive/master.zip
+      - name: Install dependencies needed to run the HuggingFace dataset example
```

Comment:
shouldn't this go in the optional dependencies?

Reply:
yes, it was old code, removed lines